cabal.project

```diff
@@ -0,0 +1,7 @@
+packages: .
+
+package ghc-tags
```

Comment:
How would flag resolution work in a released package?

Reply:
I hope that if you do:
```
cabal install -f-ghc-lib --with-compiler=ghc-9.2 ghc-tags
```
it will do the right thing, but I never needed to do that so I don't really know.

Reply:
The problem is that people are not going to do that, they're going to do `cabal install ghc-tags` and it should work out of the box, whereas it currently doesn't :thinking: